qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ServiceSteps.java

```diff
@@ -0,0 +1,99 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.page.AppointmentSchedulingPage;
+import org.openmrs.contrib.qaframework.page.ManageServiceTypesPage;
+import org.openmrs.contrib.qaframework.page.ServicePage;
+
+import io.cucumber.java.Before;
```

Comment:
Align the imports with the existing import order. Cross check the existing file and align